.buildkite/pipeline-release-helm.yml

```diff
@@ -36,7 +38,9 @@ steps:
         commands:
           - buildkite-agent artifact download "bin/releaser" /usr/local/
           - chmod u+x /usr/local/bin/releaser
-          - releaser --env=dev --excludes=eck-operator --dry-run=\$DRY_RUN
+          - releaser --env=dev --charts-dir=deploy/eck-stack --dry-run=\$DRY_RUN
```

Comment:
Do we have to allow to re-release non SNAPSHOT charts in the dev Helm repo in we want to test this?

Reply:
Yes but I would do it in another PR. I can work on it.

Reply:
> Yes but I would do it in another PR. I can work on it.

Reply:
> Yes but I would do it in another PR. I can work on it.

👍 , absolutely, sorry for not being clear. It was to be sure that this was required to test this change "for real".